instrumentation/httptrace/config.go

```diff
@@ -53,6 +55,25 @@ func newConfig() config {
 		traceClientIP:                internal.BoolEnv(envTraceClientIPEnabled, false),
 		isStatusError:                isServerError,
 		inferredProxyServicesEnabled: internal.BoolEnv(envInferredProxyServicesEnabled, false),
+		baggageTagKeys:               make(map[string]struct{}),
+	}
+	raw, ok := os.LookupEnv("DD_TRACE_BAGGAGE_TAG_KEYS")
+	if !ok {
+		// not set → default list
+		raw = "user.id,account.id,session.id"
+	}
+	// Now raw=="” only if they explicitly set it to empty
+	parts := strings.Split(raw, ",")
+	for _, part := range parts {
+		key := strings.TrimSpace(part)
+		if key == "" {
+			continue
+		}
+		if key == "*" {
```

Comment:
This means the presence of "\*" among other items in the list will toggle allowAllBaggage to true. I'd expect this behavior only if DD_TRACE_BAGGAGE_TAG_KEYS="*"